third_party/move/move-compiler-v2/legacy-move-compiler/src/parser/syntax.rs

```diff
@@ -3590,7 +3590,10 @@ fn parse_module(
                 },
                 // Regular move constructs
                 Tok::Use => ModuleMember::Use(parse_use_decl(attributes, context)?),
-                Tok::Friend if context.tokens.lookahead()? != Tok::Fun => {
+                Tok::Friend
+                    if context.tokens.lookahead()? != Tok::Fun
```

Comment:
We should probably use something like: `!vec![Tok::Fun, Tok::Inline].contains(&context.tokens.lookahead()?)` (to minimize the lookaheads). This makes even more sense given my other comment.

Reply:
done